$NetBSD: patch-Makefile.libretro,v 1.2 2018/08/08 23:47:49 nia Exp $

Don't redefine CC.

Picodrive is written in C. Let's link with a C compiler instead of
a C++ compiler.

--- Makefile.libretro.orig	2018-07-22 20:01:05.000000000 +0000
+++ Makefile.libretro
@@ -68,9 +68,6 @@ ifeq ($(platform), unix)
 	DONT_COMPILE_IN_ZLIB = 1
 	CFLAGS += -DFAMEC_NO_GOTOS
 	use_sh2drc = 1
-ifneq ($(findstring SunOS,$(shell uname -a)),)
-	CC=gcc
-endif
 
 # Portable Linux
 else ifeq ($(platform), linux-portable)
@@ -656,7 +653,7 @@ else
 	LD = link.exe
 endif
 else
-	LD = $(CXX)
+	LD = $(CC)
 endif
 
 include Makefile
